server/models/shares.js

```diff
@@ -449,7 +451,7 @@ async function regenerateRoleNamesTable() {
 
 
 function throwPermissionDenied() {
-    throw new interoperableErrors.PermissionDeniedError('Permission denied');
+    throw new interoperableErrors.PermissionDeniedError(tMark('permissionDenied'));
```

Comment:
don't work :( i also tried with tUI but i don't know how to define locales param